### PR TITLE
Bump version numbers post v0.4.28 release.

### PR DESCRIPTION
**Pull request by hawkinsp**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,9 +22,9 @@
 
 project_name = 'jax'
 
-_current_jaxlib_version = '0.4.28'
+_current_jaxlib_version = '0.4.29'
```
Comment (yashk2810) [inline setup.py:25]: This shouldn't be bumped.

Comment (yashk2810) [reply setup.py:25]: only `_latest_jaxlib_version_on_pypi`

Comment (hawkinsp) [reply setup.py:25]: When does that get bumped?

**Review by yashk2810 — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (yashk2810) [reply setup.py]: When you start the release. It's the first step.

**Pull request closed without merging**